accepted/future-releases/horizontal-inference/feature-specification.md

```diff
@@ -209,8 +220,9 @@ Performing type inference on an invocation consists of the following steps:
    try to match the return type of the target function type as a subtype of the
    invocation's type context.  This produces an initial set of type constraints.
    Then, using those constraints, find the constraint solution for the target
-   function type's type variables, considering bounds.  This produces a
-   preliminary mapping of type variables to type schemas.
+   function type's type variables, using a previous mapping that maps all type
```

Comment:
As below, I find the terminology "previous mapping" confusing here. I would say "using an initial partial solution that maps all type variables..."

Reply:
Done